app/pathogen/arbovirus/dashboard/(map)/MapArbovirusFilter.tsx

```diff
@@ -100,8 +100,8 @@ export const MapArbovirusFilter = ({ records, className }: MapArbovirusFilterPro
             </div>
           );
         })}
-        <Button className="mt-2" variant={'ghost'} onClick={clearAllHandler}>Clear all</Button>
-      </CardContent>
-    </Card>
+        <Button className="mt-2" variant={'ghost'} onClick={clearAllHandler}>Clear all viruses</Button>
+      </div>
+    </div>
```

Comment:
I have left this button in for now, but if requested I can delete it. I think it kind of makes sense to leave it in?

Reply:
yeah lets leave it in